django_lightweight_queue/app_settings.py

```diff
@@ -21,3 +21,7 @@ def setting(suffix, default):
 REDIS_HOST = setting('REDIS_HOST', '127.0.0.1')
 REDIS_PORT = setting('REDIS_PORT', 6379)
 REDIS_PREFIX = setting('REDIS_PREFIX', '')
+
+ENABLE_PROMETHEUS = setting('ENABLE_PROMETHEUS', True)
```

Comment:
> It's naturally a shame if consumers of this library don't need prometheus, or get pwned because they haven't set up firewalls, but I don't think it's sensible to make that the overriding concern - we should be trying to make a library that is as easy for us to use as possible and hope that other people find it useful.

Ok, I think this is possibly the crux of where we differ -- I don't feel that this is a good approach for an open source repo. (I think I'd still raise the "defaults to opening an unexpected port" being a bad thing for an internal library, but it's easier to drop in that scenario).

I've just realised that this repo doesn't have a README, so I'll add a brief one which highlights that this is enabled by default.